models/migrations/v161.go

```diff
@@ -34,7 +34,7 @@ func convertTaskTypeToString(x *xorm.Engine) error {
 	}
 
 	type HookTask struct {
-		Typ string `xorm:"char(16) index"`
+		Typ string `xorm:"VARCHAR(16) index"`
```

Comment:
Should we modify prior migrations?

Reply:
Yes, this will help not to break them in the first place and help in case when you are migrating from v1.13.x to v1.14.0

Reply:
How exactly will it help? The v165 will run anyway.

Reply:
It will let v165 faster because once database know the type of the modify column is the same as before, it will ignore that.